control-plane/plugin/src/operations.rs

```diff
@@ -64,5 +80,4 @@ pub trait Cordoning {
     type ID;
     async fn cordon(id: &Self::ID, label: &str, output: &utils::OutputFormat);
     async fn uncordon(id: &Self::ID, label: &str, output: &utils::OutputFormat);
-    async fn get_labels(id: &Self::ID, output: &utils::OutputFormat);
```

Comment:
Why do we want to remove this? Can you please update the [README](https://github.com/openebs/mayastor-control-plane/blob/develop/k8s/plugin/README.md) with the examples and usages.

Reply:
After talking with Tiago, it seemed better to implement a new "get cordon" command in the kubectl plugin to replace the --get-cordon-labels flag associated with get node. It makes the code more elegant and more obvious for the user.
I will update the readme in a final PR but I think there may be other changes before the final PR (which is why this is still a draft PR) e.g. Tiago would like to replace cordon_labels + drain_labels + is_draining with an enum variant. I am still looking into that.

Reply:
Okay.